src/data/moves_info.h

```diff
@@ -154,7 +156,8 @@ const struct MoveInfo gMovesInfo[MOVES_COUNT_DYNAMAX] =
         .contestEffect = CONTEST_EFFECT_HIGHLY_APPEALING,
         .contestCategory = CONTEST_CATEGORY_TOUGH,
         .contestComboStarterId = COMBO_STARTER_POUND,
-        .contestComboMoves = {0}
+        .contestComboMoves = {0},
+        .battleAnimScript = Move_POUND,
```

Comment:
I assume you have a script that moved it over. Would it be feasible to move the animscript above the contest data so it stays isolated?

Also assuming you had a script could you provide it so users can use it for their custom moves?

Reply:
> I assume you have a script that moved it over. Would it be feasible to move the animscript above the contest data so it stays isolated?  

Would need to change quite a bit, due to contestComboMoves not always being the last entry and additionalEffects (sometimes?) taking up multiple lines.
> Also assuming you had a script could you provide it so users can use it for their custom moves?  

How/Where do you want me to share it?

Reply:
> How/Where do you want me to share it?

There is a folder with migration scripts.

> Would need to change quite a bit, due to contestComboMoves not always being the last entry and additionalEffects (sometimes?) taking up multiple lines.

Yeah, I see. I personally don't mind if they are at the end but some others might disagree so it would be good to get some additional input.

Reply:
Pushed the script, not sure if thats what you want, since its basically tailored for the current state of the file. It does not fix battle_anim_scripts.s since there are some scripts that have names that are inconsistent with the move definitions. E.g. MOVE_FEINT_ATTACK vs Move_FAINT_ATTACK

Reply:
> Pushed the script, not sure if thats what you want, since its basically tailored for the current state of the file. It does not fix battle_anim_scripts.s since there are some scripts that have names that are inconsistent with the move definitions. E.g. MOVE_FEINT_ATTACK vs Move_FAINT_ATTACK

Thanks. Being tailored to the repo is fine since we can't know what users did.